src/components/AddEvent.tsx

```diff
@@ -111,10 +119,29 @@ const AddEvent = () => {
           type="text"
         />
       </Form.Group>
-
       <Form.Group>
         <Form.Label>Date:</Form.Label>
-        <Form.Control type="date" value={date} onChange={dateChangeHandler} />
+        <DateInput />
+      </Form.Group>
+      <Form.Group>
+        <Form.Check
```

Comment:
use radio buttons if only 1 option will be available.
use two-way binding (have the value prop(the one you keep in the state) passed to the input)